src/utils/addresses.ts

```diff
@@ -73,3 +77,17 @@ export const sortAddressList = (addresses: Address[]): Address[] =>
     if (b.settings.isMain) return 1
     return (b.lastUsed ?? 0) - (a.lastUsed ?? 0)
   })
+
+export const migrateAddressMetadata = () => {
+  const Storage = getStorage()
+  const walletNames = Storage.list()
+
+  for (const name of walletNames) {
+    const data = localStorage.getItem(`${name}-${addressesMetadataLocalStorageKeyPrefix}`)
```

Comment:
I think this could use a comment - seeing something with the description of prefix being used as a suffix is very confusing.

Reply:
The old prefix / name should be local to the migration's function context.

Reply:
OK, how about now? :)